pgxn/neon/logical_replication_monitor.c

```diff
@@ -20,18 +21,31 @@
 #define LS_MONITOR_CHECK_INTERVAL 10000 /* ms */
 
 static int	logical_replication_max_snap_files = 300;
+static int	logical_replication_max_logicalsnapdir_size = 128;
```

Comment:
given the recent improvements of the aux file scan path, we could probably set a larger limit (i.e., 8MB?)

Reply:
How did you come up with 8MB? Just want to make sure the value is documented with some reasoning.

Reply:
Pageserver should be okay with 1KB * 10000 snapshot files based on my previous observations; floor to 8MB which is power of 2

Reply:
@skyzh please double check the comment I added in my latest push.